students/Sahlberg/Lesson3/mailroom.py

```diff
@@ -0,0 +1,90 @@
+"""Ian Sahlberg
+Assignment 3 mailroom
+Python 210
+12/28/2018"""
+
+
+#Disctionary of donors name, dollar amount
+donor = {'bob johnson': [150.00],
+         'susan skoosan': [2000.00, 550.00],
+         'tim tam':[10.50],
+         'roxanne raffle':[13.00, 75.00, 123.00],
+         'jon jacob':[5000.00, 5.00]
+         }
+
+def donate(name, dictionary):
+    """Appends a donation amount to a given donor."""
+    amount = int(input('How much did they donate?'))
+    return amount, dictionary[name].append(amount)
+
+def print_list(dictionary):
+    """Prints our the donor list"""
+    return print(list(dictionary.keys()))
+
+def user_options():
+    """Main user options menu choices."""
+    user_action = input('Please choose from the following options: Send a Thank You, Create a Report, quit')
```

Comment:
Recommend not using 'quit' as it's a reserved word.  Can make things a bit complicated.